src/Microsoft.DocAsCode.Build.OverwriteDocuments/OverwriteUtility.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.OverwriteDocuments
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text.RegularExpressions;
+
+    public class OverwriteUtility
+    {
+        private static readonly Regex OPathRegex =
+            new Regex(
+                @"^(?<propertyName>[:A-Za-z_][\w\.\-:]*)(\[(?<key>[:A-Za-z_][\w\.\-:]*)=((""(?<stringValue>[\w\(\)\.\{\}\[\]\|\/@ `<>:]+)"")|(?<numberValue>[-+]?[0-9]*(\.[0-9]+)?))\])?/?",
```

Comment:
>stringValue [](start = 95, length = 11)

is boolean supported, e.g. parameters[enable=true]

Reply:
yes, it should be supported

---
In reply to: [162852574](https://github.com/dotnet/docfx/pull/2363#discussion_r162852574) [](ancestors = 162852574)

Reply:
``^\s*(?<propertyName>[:A-Za-z_][\w\.\-:]*)\s*(?:\[\s*(?<key>[:A-Za-z_][\w\.\-:]*)\s*=\s*(?:(?:"(?<stringValue>[\w\(\)\.\{\}\[\]\|\/@ `<>:]+)")|(?<numberValue>[-+]?[0-9]*(\.[0-9]+)?))\s*\])?\s*(?:\/|$)``